modules/katamari/src/test/ts/atomic/api/str/ContainsTest.ts

```diff
@@ -52,4 +52,21 @@ describe('atomic.katamari.api.str.ContainsTest', () => {
       (value) => !Strings.contains('', value)
     ));
   });
+
+  it('should handle the start and the end parameters', () => {
+    const check = (expected: boolean, str: string, substr: string, start: number, end?: number) => {
+      const actual = Strings.contains(str, substr, start, end);
+      assert.equal(actual, expected);
+    };
+
+    check(true, 'abc', 'b', 0);
+    check(true, 'abc', 'b', 1);
+    check(false, 'abc', 'b', 2);
+    check(false, 'abc', 'b', 0, 0);
+    check(true, 'abc', 'b', 0, 2);
+    check(true, 'abc', 'b', 1, 3);
+    check(false, 'abc', 'b', 2, 3);
+    check(false, 'abc', 'b', 0, 1);
```

Comment:
please add a test for cases where the `substr` parameter is longer than the remaining string

for example `check(false, 'abc', 'bc', 1, 2);` should do it?